DNN Platform/Library/Services/FileSystem/FileManager.cs

```diff
@@ -899,7 +899,7 @@ public virtual IFileInfo GetFile(int fileID, bool retrieveUnpublishedFiles)
             var file = DataCache.GetCache(strCacheKey);
             if (file == null)
             {
-                file = CBO.Instance.FillObject<FileInfo>(DataProvider.Instance().GetFileById(fileID, retrieveUnpublishedFiles));
```

Comment:
This is a bad example of wsing"CBO.Instance.XXX"; there were only 6 places in the whole Platform source code that use CBO as an instance and it is used as a static object everywhere else (over 420 places). When we use the instance, it must be either all over the place or none as there is no meaning to use it this way in one place and write unit tests for this single case and leaving git all around the place as static object.